server/events.go

```diff
@@ -88,20 +90,32 @@ type ServerStatsMsg struct {
 
 // ConnectEventMsg is sent when a new connection is made that is part of an account.
 type ConnectEventMsg struct {
+	Schema string     `json:"schema"`
+	ID     string     `json:"id"`
+	Time   string     `json:"timestamp"`
 	Server ServerInfo `json:"server"`
 	Client ClientInfo `json:"client"`
 }
 
+// ConnectEventMsgSchema is the schema ID for ConnectEventMsg
+const ConnectEventMsgSchema = "io.nats.server.advisory.v1.client_connect"
```

Comment:
I put the constants by the struct definition so its easy to find, don't want to add files and do a bunch of new things so optimise for ease of discovery while reading code for the moment.